slog/slog.go

```diff
@@ -170,43 +160,29 @@ func With(args ...any) *Logger {
 	return slog.With(args...)
 }
 
-type tracedHandler struct {
-	handler slog.Handler
-}
-
-func (t *tracedHandler) Enabled(ctx context.Context, l slog.Level) bool {
-	return t.handler.Enabled(ctx, l)
-}
-
-func (t *tracedHandler) Handle(ctx context.Context, record slog.Record) error {
-	if traceID, ok := getValue(ctx, TraceID); ok {
-		record.Add("trace_id", traceID)
-	}
-	if orgID, ok := getValue(ctx, OrgID); ok {
-		record.Add("organization_id", orgID)
-	}
-	return t.handler.Handle(ctx, record)
-}
-
-func getValue(ctx context.Context, key any) (string, bool) {
-	val := ctx.Value(key)
-	if val == nil {
-		return "", false
-	}
-	str, ok := val.(string)
+// FromCtx gets the [Logger] associated with the given context. An empty [Logger] is
+// returned if the context has no [Logger] associated with it.
```

Comment:
We don't really return an empty log but the default one from slog, someone could update before for example. My point is just fix the comment, saying that if empty slog.Default() will be returned